src/utils/src/Allocators.c

```diff
@@ -86,4 +89,4 @@ BOOL checkBufferValues(PVOID ptr, BYTE val, SIZE_T size)
     return TRUE;
 }
 
-memChk globalMemChk = checkBufferValues;
\ No newline at end of file
+memChk globalMemChk = checkBufferValues;
```

Comment:
New line after this.

Reply:
I believe it's the diffing tool issue - the code has the trailing \n at the bottom of the file